pkg/probe/log_fortianalyzer_queue.go

```diff
@@ -0,0 +1,55 @@
+package probe
+
+import (
+	"log"
+
+	"github.com/bluecmd/fortigate_exporter/pkg/http"
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+type LogAnaQueueResults struct {
+	Connected  int `json:"connected"`
+	FailedLogs int `json:"failed_logs"`
+	CachedLogs int `json:"cached_logs"`
```

Comment:
```suggestion
	Connected  float64 `json:"connected"`
	FailedLogs float64 `json:"failed_logs"`
	CachedLogs float64 `json:"cached_logs"`
```
Let the parser directly convert to float64